### PR TITLE
changed my booking icon in facility calendar page

### PR DESCRIPTION
**Pull request by limwenfeng**
image

image

### CODE REVIEW

File: frontend/src/routes/FacilityBooking/ViewFacility/index.tsx [hunk 1]
```diff
@@ -40,14 +40,11 @@ export default function ViewFacility() {
...
-      state="primary"
-      text="Bookings"
-      type="button"
-      size="small"
+  const myBookingsIcon = (
+    <img
+      src={MyBookingsIcon}
       onClick={() => {
-        history.push(`${PATHS.VIEW_MY_BOOKINGS_USERID}/${localStorage.getItem('userID')}`)
+        history.push(PATHS.VIEW_MY_BOOKINGS_USERID + '/' + localStorage.getItem('userID'))
```
Comment (hui444) [inline frontend/src/routes/FacilityBooking/ViewFacility/index.tsx:47]: why was this updated?

**Review by hui444 — COMMENTED** [review]
icons were taken from antd; pls import it instead of using svg!

File: frontend/src/routes/FacilityBooking/ViewFacility/index.tsx (hunk 1 above)
Comment (limwenfeng) [reply frontend/src/routes/FacilityBooking/ViewFacility/index.tsx:47]: eh what the. i didnt touch this. okay i change back

File: frontend/src/routes/FacilityBooking/ViewFacility/index.tsx [hunk 2]
```diff
@@ -40,12 +40,9 @@ export default function ViewFacility() {
     }
   }, [])
 
-  const MyBookingButton = (
-    <Button
-      state="primary"
-      text="Bookings"
-      type="button"
-      size="small"
+  const myBookingsIcon = (
```
Comment (rldefa) [inline frontend/src/routes/FacilityBooking/ViewFacility/index.tsx:43]: I feel that it is better to maintain naming scheme
```suggestion
  const MyBookingsIcon = (
```

File: frontend/src/routes/FacilityBooking/ViewBookingDailyView/index.tsx [hunk 3]
```diff
@@ -63,19 +63,19 @@ export default function ViewBookingDailyView() {
     }
   }, [date])
 
+  const bookingFacilityIcon = (
```
Comment (rldefa) [inline frontend/src/routes/FacilityBooking/ViewBookingDailyView/index.tsx:66]:
```suggestion
  const BookingFacilityIcon = (
```

File: frontend/src/routes/FacilityBooking/ViewBookingDailyView/index.tsx [hunk 4]
```diff
@@ -63,19 +63,19 @@ export default function ViewBookingDailyView() {
...
+    <UnorderedListOutlined
+      style={{ color: 'black', fontSize: '150%' }}
+      onClick={() => console.log('link to booking page')}
```
Comment (rldefa) [inline frontend/src/routes/FacilityBooking/ViewBookingDailyView/index.tsx:69]: What is stopping us from linking it now?

Comment (limwenfeng) [reply frontend/src/routes/FacilityBooking/ViewBookingDailyView/index.tsx:69]: I think the revamp booking page not done?

File: frontend/src/routes/FacilityBooking/ViewBookingDailyView/index.tsx (hunk 3 above)
Comment (limwenfeng) [reply frontend/src/routes/FacilityBooking/ViewBookingDailyView/index.tsx]: Fixed #[5fbad19](https://github.com/rhdevs/rhapp/pull/639/commits/5fbad19aa74a40cdc7965b02d74c8f8d0498242a)

File: frontend/src/routes/FacilityBooking/ViewFacility/index.tsx (hunk 2 above)
Comment (limwenfeng) [reply frontend/src/routes/FacilityBooking/ViewFacility/index.tsx]: Fixed in #[5fbad19](https://github.com/rhdevs/rhapp/pull/639/commits/5fbad19aa74a40cdc7965b02d74c8f8d0498242a)

**Review by rldefa — APPROVED** [review]
lgtm

**Pull request merged**